qt/scientific_interfaces/Inelastic/test/Analysis/FqFitDataPresenterTest.h

```diff
@@ -63,25 +64,30 @@ class FqFitDataPresenterTest : public CxxTest::TestSuite {
     m_presenter = std::make_unique<FqFitDataPresenter>(m_tab.get(), m_model.get(), m_view.get());
     m_workspace = createWorkspaceWithTextAxis(6, getTextAxisLabels());
     m_ads = std::make_unique<SetUpADSWithWorkspace>("WorkspaceName", m_workspace);
+    m_fitPropertyBrowser = std::make_unique<NiceMock<MockFitPropertyBrowser>>();
+    m_presenter->subscribeFitPropertyBrowser(std::move(m_fitPropertyBrowser.get()));
```

Comment:
In this case the `std::move` doesn't do anything because we are passing in a raw non-owning pointer. Plus we also don't want the presenter to own the browser, just have a pointer to it
https://stackoverflow.com/questions/3413470/what-is-stdmove-and-when-should-it-be-used

Reply:
I'm trying to do that in the latest commit, but if `m_fitpropertybrowser` is not unique pointer, the test throws an exception, do you know why?

Reply:
I think we want to keep `m_fitPropertyBrowser` as a unique pointer, but on the subscribe line we want to do:
```
m_presenter->subscribeFitPropertyBrowser(m_fitPropertyBrowser.get());
```
(i.e. without the `std::move`) because we don't want the presenter to own the browser. Instead we want the TestSuite to own the browser. Does this work?

Reply:
No, it seems it doesn't work with that syntax either.